tutorials/rendering/jitter_stutter.rst

```diff
@@ -241,6 +459,127 @@ If your mouse offers multiple :abbr:`DPI (Dots Per Inch)` settings, consider als
+    The question is, how much FPS (frames per second) are you willing to sacrifice for latency. The relationship is not linear,
+    which means the lower the latency you want to achieve the greater the FPS sacrifice, which can get ridiculously high.
+    This is not a bug, it is just the nature of how it works.
+
+.. warning::
+
+    Lowering latency makes Godot more susceptible to **microstutter**.
+    Thus if you're experiencing microstutter, use these options to *increase* latency instead.
+
+.. warning::
+    ``low_extreme`` might get you better latency (or even be a placebo) but always at a very large FPS cost.
+    Furthermore it may reintroduce microstutter. Caution is advised when seeking to lower latency too much.
+
+.. warning::
+    Just because ``low_extreme`` works great in your machine doesn't mean it will work fine
+    in other machines. Don't deploy to end users with this setting as a default.
+
+The following table summarizes what Godot does based on each setting and available feature:
+
+.. table::
+   :widths: auto
+
+   +-----------------+-----------------------------------------------------------------+---------------------------------+
+   | Setting         | Target: Frames of latency (if Waitable Swapchains are available)| Waitable Swapchains Unavailable |
+   +=================+=================================================================+=================================+
+   | low_extreme     | 0                                                               | SEQUENTIAL                      |
+   +-----------------+-----------------------------------------------------------------+---------------------------------+
+   | low             | 1                                                               | AUTO                            |
+   +-----------------+-----------------------------------------------------------------+---------------------------------+
+   | medium          | rendering/rendering_device/vsync/frame_queue_size               | PARALLEL (Disabled)             |
+   +-----------------+-----------------------------------------------------------------+---------------------------------+
+   | high_throughput | Disabled                                                        | PARALLEL (Disabled)             |
+   +-----------------+-----------------------------------------------------------------+---------------------------------+
+
+This table is read like this:
+ 
+ - If Waitable Swapchains are available, at "low" Godot will target 1 frame of latency. Otherwise, at "low" it will use AUTO as fallback.
+ - If Waitable Swapchains are available, at "medium" Godot will use the value in frame_queue_size  as target for frames of latency. Otherwise it uses PARALLEL mode (which is the same as having no pacing method).
+
+
+.. tip::
+
+    Godot's behavior on version 4.4 and earlier corresponds to ``high_throughput``.
```

Comment:
```suggestion
This table is read like this:
 
 - If Waitable Swapchains are available, at "low", Godot will target 1 frame of latency. Otherwise, at "low" it will use AUTO as fallback.
 - If Waitable Swapchains are available, at "medium" Godot will use the value in ``frame_queue_size`` as target for frames of latency. Otherwise, it uses ``PARALLEL`` mode (which is the same as having no pacing method).

.. tip::

    Godot's behavior on version 4.4 and earlier corresponds to ``high_throughput``.
```